appveyor.yml

```diff
@@ -1,15 +1,27 @@
 version: 0.0.0.{build}
 os: Default Azure
 configuration:
-- Debug
 - Release_40
 - Release_45
 shallow_clone: true
+matrix: 
+    fast_finish: true
+before_test:
+  - ps: Start-FileDownload 'http://garr.dl.sourceforge.net/project/firebird/firebird-win32/2.5.4-Release/Firebird-2.5.4.26856_0_Win32.exe' | Out-Null
```

Comment:
Maybe download the zip and just unpack and run? It might be faster than installation.